pal/src/map/common.cpp

```diff
@@ -49,7 +49,7 @@ INT W32toUnixAccessControl( IN DWORD flProtect )
         MemAccessControl = PROT_EXEC | PROT_READ | PROT_WRITE;
         break;
     case PAGE_EXECUTE :
-        MemAccessControl = PROT_EXEC;
+        MemAccessControl = PROT_EXEC | PROT_READ; // WinAPI PAGE_EXECUTE also implies readable
```

Comment:
The goal here is to mimic windows behavior.  I believe @litian2025  has experiement that `PAGE_EXECUTE` gives you read permission.    What do you mean "limited" READ?   I though reading is reading is reading.   How do you distinguish different "kind" of reading?